ini/stm32f4.ini

```diff
@@ -806,3 +806,20 @@ board_build.offset = 0x8000
 build_flags        = ${stm32_variant.build_flags} -DSTM32F407_5ZX
 debug_tool         = stlink
 upload_protocol    = stlink
+
+#
+# Mellow Fly E3 V2 (STM32F407VGT6 ARM Cortex-M4)
+#
+[env:FLY_E3_V2]
+extends                     = stm32_variant
+platform_packages           = ${stm_flash_drive.platform_packages}
+board                       = marlin_STM32F407VGT6_CCM
+board_build.variant         = MARLIN_F4x7Vx
+board_build.offset          = 0x8000
+board_upload.offset_address = 0x08008000
+build_flags                 = ${stm_flash_drive.build_flags}
```

Comment:
Since this board doesn't support flash drives, this build flag should be removed.